practices/practice-one/src/scripts/controllers/student.controller.js

```diff
@@ -3,6 +3,29 @@ import { axiosClient } from '../helpers/utils.js';
 export default class Controller {
     constructor() {}
 
+    async getStudents() {
```

Comment:
Can we move this as a helper then re-use?

Reply:
I don't know yet, but my intention is to use it to get all students to use in render